datasets/kinnews_kirnews/kinnews_kirnews.py

```diff
@@ -0,0 +1,134 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Kinyarwanda and Kirundi news classification datasets."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import datasets
+
+
+_CITATION = """\
+@article{niyongabo2020kinnews,
+  title={KINNEWS and KIRNEWS: Benchmarking Cross-Lingual Text Classification for Kinyarwanda and Kirundi},
+  author={Niyongabo, Rubungo Andre and Qu, Hong and Kreutzer, Julia and Huang, Li},
+  journal={arXiv preprint arXiv:2010.12174},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+Kinyarwanda and Kirundi news classification datasets
+"""
+
+_HOMEPAGE = "https://github.com/Andrews2017/KINNEWS-and-KIRNEWS-Corpus"
+_LICENSE = "MIT License"
+
+_URLs = {
+    "kinnews": "https://github.com/saradhix/kinnews_kirnews/raw/master/KINNEWS.zip",
+    "kirnews": "https://github.com/saradhix/kinnews_kirnews/raw/master/KIRNEWS.zip",
+}
+
+
+class KinnewsKirnews(datasets.GeneratorBasedBuilder):
+    """This is Kinyarwanda and Kirundi news dataset called KINNEWS and KIRNEWS."""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(name="kinnews_raw", description="Dataset for Kinyarwanda language"),
+        datasets.BuilderConfig(name="kinnews_cleaned", description="Cleaned dataset for Kinyarwanda language"),
+        datasets.BuilderConfig(name="kirnews_raw", description="Dataset for Kirundi language"),
+        datasets.BuilderConfig(name="kirnews_cleaned", description="Cleaned dataset for Kirundi language"),
+    ]
+
+    def _info(self):
+        if "raw" in self.config.name:
+            features = datasets.Features(
+                {
+                    "label": datasets.Value("int32"),
```

Comment:
Can you use the ClassLabel feature type here please ?

For example
```
"label": datasets.ClassLabel(names=["politic", "sport", etc.])
```

Also the label ids must start at zero, however it looks like the first label is politic with label `1`. So you would need to do `label = label - 1`

Reply:
Yes, used ClassLabel with descriptive names.